File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewMyStandardControlsRow.cpp

#include "RowBasedCommand_ViewMyStandardControlsRow.h"
#include "RowBasedCommand_ViewMyStandardControlsRow.h"
#include <string>

namespace commands
{
  std::string RowBasedCommand_ViewMyStandardControlsRow::getTextBoxesTextBoxText( ) 
  {
    return this->TextBoxesTextBoxText;
  }
  int RowBasedCommand_ViewMyStandardControlsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
  int RowBasedCommand_ViewMyStandardControlsRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewMyStandardControlsRow.cs
/// <filename>
///     RowBasedCommand_ViewMyStandardControlsRow.cs
/// </filename>


namespace commands
{
    internal class RowBasedCommand_ViewMyStandardControlsRow
  {
        private string TextBoxesTextBoxText;
     
        public     virtual string getTextBoxesTextBoxText()
    {
      return this.TextBoxesTextBoxText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

